### ISSUE TITLE
Style/RescueEnsureAlignment error

### ISSUE THREAD

**Issue by daviddavis** [bbatsov/rubocop#2171]
Rubocop seems to error on this code:

```ruby
begin require 'pry'; rescue LoadError; nil end
```

I know it's bad style. Here's the stack trace:

```
undefined method `include?' for nil:NilClass
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/cop/style/rescue_ensure_alignment.rb:68:in `modifier?'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/cop/style/rescue_ensure_alignment.rb:28:in `on_resbody'
(eval):5:in `block (2 levels) in on_resbody'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/cop/commissioner.rb:92:in `with_cop_error_handling'
(eval):4:in `block in on_resbody'
(eval):2:in `each'
(eval):2:in `on_resbody'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:259:in `process'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:276:in `block in process_all'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:275:in `map'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:275:in `process_all'
/Users/david/.rvm/gems/ruby-2.2.1/gems/parser-2.2.2.6/lib/parser/ast/processor.rb:9:in `process_regular_node'
(eval):9:in `on_rescue'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:259:in `process'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:276:in `block in process_all'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:275:in `map'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:275:in `process_all'
/Users/david/.rvm/gems/ruby-2.2.1/gems/parser-2.2.2.6/lib/parser/ast/processor.rb:9:in `process_regular_node'
(eval):9:in `on_kwbegin'
/Users/david/.rvm/gems/ruby-2.2.1/gems/ast-2.1.0/lib/ast/processor/mixin.rb:259:in `process'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/cop/commissioner.rb:52:in `investigate'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/cop/team.rb:33:in `inspect_file'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:135:in `inspect_file'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:106:in `block in do_inspection_loop'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:100:in `loop'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:100:in `do_inspection_loop'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:78:in `process_file'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:53:in `block in inspect_files'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:51:in `each'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:51:in `inspect_files'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/runner.rb:30:in `run'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/lib/rubocop/cli.rb:26:in `run'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/bin/rubocop:13:in `block in <top (required)>'
/Users/david/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/benchmark.rb:303:in `realtime'
/Users/david/.rvm/gems/ruby-2.2.1/gems/rubocop-0.33.0/bin/rubocop:12:in `<top (required)>'
/Users/david/.rvm/gems/ruby-2.2.1/bin/rubocop:23:in `load'
/Users/david/.rvm/gems/ruby-2.2.1/bin/rubocop:23:in `<main>'
/Users/david/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `eval'
/Users/david/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `<main>'
```

Rubocop version: 0.33.0
Ruby version: ruby 2.2.1p85 (2015-02-26 revision 49769) [x86_64-darwin14]

Reproducer:
```bash
git clone https://github.com/daviddavis/rubocop_rescue_test.git
cd rubocop_rescue_test
bundle install
rubocop --config .rubocop_todo.yml
```

**Comment by bquorning** [comment]
This seems to be a duplicate of #2119, fixed in master by #2137.

**Comment by daviddavis** [comment]
Yes, tested and it works. I could have sworn I searched but guess not.
